### PR TITLE
Add vintage U.S. Route shield for historic routes

### LINKED ISSUE #921: Historic U.S. Route shield is too subtle
The shield for Historic U.S. Routes is identical to the usual U.S. Route shield but recolored from black to brown. This is too subtle against an off-white background, especially in some color profiles.

[County Route 10 and Historic U.S. Route 66](https://zelonewolf.github.io/openstreetmap-americana/#map=13/35.1172/-114.14085)

We’ve been getting feedback in [OSMUS Slack](https://osmus.slack.com/archives/C01V02K52UX/p1694307360310139) about this shield causing confusion:

> Another one is that US-80 has been decommissioned for decades now. There are a number of relations for the historical route US-80 which apparently are being rendered as a still commissioned route. Would it be possible to have a historical route shield for the historical routes?

### PR DESCRIPTION
**Pull request by 1ec5**
Added a vaguely 1948-era U.S. Route shield for Historic U.S. Routes. This shield is the same as the one we’ve been using (just a modern U.S. Route shield recolored from black to brown), but with a crossbar up top, similar to what we’ve done with the Maryland state route shield.

Here’s a before and after:

[Before: Arizona State Route 66 and Historic U.S. Route 66 at Interstate 40 and U.S. Route 93](https://zelonewolf.github.io/openstreetmap-americana/#map=13/35.22067/-114.00642) [After: Arizona State Route 66 and Historic U.S. Route 66 at Interstate 40 and U.S. Route 93](https://zelonewolf.github.io/openstreetmap-americana/#map=13/35.22067/-114.00642)

This treatment is consistent with many states’ signs for these historic routes. Some states go further back to the 1926 style with a blocky typeface, but we haven’t been tailoring shields to each country’s typefaces anyways. Originally, I had considered adding a “HIST” banner at the top, but I think the crossbar is distinctive enough.

[Historic U.S. 165 in Louisiana](https://commons.wikimedia.org/wiki/File:US_165_Historic_Route.JPG)

[Historic U.S. 66 in Arizona](https://commons.wikimedia.org/wiki/File:09_Seligman_2019.jpg)

Fixes #921.

### CODE REVIEW

**Comment by zekefarwell** [comment]
Looks great!  

> Originally, I had considered adding a “HIST” banner at the top, but I think the crossbar is distinctive enough.

I think this is a good idea.  For those of us in the know, the brown and the crossbar is distinctive enough, but for those who spend less time thinking about highway shields it's probably not clear what the different style means.  Perhaps the legend is enough to answer that question for folks, but a HIST banner would make it clearer and would match the "historic" wording that is generally on the signs.

**Comment by ZeLonewolf** [comment]
I think the retro shield looks great but it doesn't solve the problem of the brown being too subtle. So I'd say we either need a banner or some way to make the brown be more obvious.

**Comment by jleedev** [comment]
White text on a brown shield? It would not be entirely unrealistic.

**Comment by 1ec5** [comment]
> I think this is a good idea. For those of us in the know, the brown and the crossbar is distinctive enough, but for those who spend less time thinking about highway shields it's probably not clear what the different style means. Perhaps the legend is enough to answer that question for folks, but a HIST banner would make it clearer and would match the "historic" wording that is generally on the signs.

I think we’re basically at the point of the difference between an Interstate and an Interstate Business Loop/Spur, where the user will at least notice there’s something unusual about the shield, cuing them to consult the legend. That said, I’m still open to adding a “HIST” banner. It’s just that we only support drawing the banner in black at the moment, which is kind of awkward.

[HIST banner: Arizona State Route 66 and Historic U.S. Route 66 at Interstate 40 and U.S. Route 93](https://zelonewolf.github.io/openstreetmap-americana/#map=13/35.22067/-114.00642)

> White text on a brown shield? It would not be entirely unrealistic.

I like how this would look visually, but it does take some liberties with the shield, which is white in every state.

[White on brown: Arizona State Route 66 and Historic U.S. Route 66 at Interstate 40 and U.S. Route 93](https://zelonewolf.github.io/openstreetmap-americana/#map=13/35.22067/-114.00642)

Note that the former U.S. Route 412 Scenic in Oklahoma was also set against a brown sign back. Idaho also used to signpost the Thousand Islands Scenic Byway [concurrency](https://www.openstreetmap.org/way/142994987) with U.S. 30 (not U.S. 30 Scenic) as [a white-on-brown shield](http://www.aaroads.com/shields/show.php?image=ID19790843&view=4), though ITD has since [replaced it](https://www.bing.com/maps?osid=b997cc64-0290-4f2b-bd7b-5a8e62a47440&cp=42.934467~-114.988675&lvl=17&dir=174.47498&pi=0.16404386&style=x&v=2&sV=2&form=S00027) with a standard U.S. Route shield.

**Comment by ZeLonewolf** [comment]
> I like how this would look visually, but it does take some liberties with the shield, which is white in every state.

While the white on brown may not be perfectly accurate, it very nicely communicates the familiar feel of the historic shield.

**Comment by zekefarwell** [comment]
White on brown looks nice (reminds me of the USFS shields), but given how closely this style replicates real world shields the brown on white seems the better choice to me.  

> I’m still open to adding a “HIST” banner. It’s just that we only support drawing the banner in black at the moment, which is kind of awkward.

Good point.  A brown banner would look better.  Perhaps adding color support for banners can be a future improvement.  There are other shields where the banner matching the primary color would look better and be more accurate. Green on Vermont shields for example:

Screenshot 2023-09-10 at 3 10 05 PM

Screenshot 2023-09-10 at 3 12 23 PM

**Comment by ZeLonewolf** [comment]
> Good point. A brown banner would look better. Perhaps adding color support for banners can be a future improvement. There are other shields where the banner matching the primary color would look better and be more accurate. Green on Vermont shields for example:

Agreed.  This would be a welcome feature for the shield library.  Currently it only accepts a global default color, but what's needed is a per-network override option.  I suspect it would be relatively easy to implement.

**Comment by 1ec5** [comment]
> A brown banner would look better. Perhaps adding color support for banners can be a future improvement. There are other shields where the banner matching the primary color would look better and be more accurate. Green on Vermont shields for example:

#357 tracks drawing banners in colors other than black. In the meantime, I’ve added the “HIST” banner in black.

File: icons/shield_us_us_historic_2.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg width="20" height="20" xmlns="http://www.w3.org/2000/svg">
```
Comment (claysmalley) [inline icons/shield_us_us_historic_2.svg:1]: These icons might have a generic use in some Latin American countries. Consider a more generic filename like `badge_crossbar`.

Comment (1ec5) [reply icons/shield_us_us_historic_2.svg:1]: Done. shield_badge_crossbar_{2,3}.svg is now black on white like shield_badge_{2,3}.svg, and this network’s definition recolors it to brown.

When we get around to implementing Bolivia’s federal routes and Paraguay’s primary routes, we’ll be able to recolor these images to white on green. Some other countries have similar-looking shields that aren’t similar enough to reuse these images:

* Chile’s national routes are shaped differently at the top: https://github.com/ZeLonewolf/openstreetmap-americana/pull/474#discussion_r913828239.
* The Pan-American Highway shield is inspired by the old U.S. Route shield, but it’s defaced with a map of the Americas: https://github.com/ZeLonewolf/openstreetmap-americana/issues/421#issuecomment-1159569738.
* Peru’s shields are stretched vertically.

**Review by claysmalley — APPROVED** [review]
I was on the fence about including both the crossbar and the "HIST" banner, but seeing it in context really hammers it home. Looks good to me.

Example location ([production](https://americanamap.org/#map=10.2/35.2184/-101.7296), [localhost](http://localhost:1776/#map=10.2/35.2184/-101.7296)):

image

**Pull request merged**